spacq/devices/keithley/dm2100.py

```diff
@@ -0,0 +1,119 @@
+import logging
+log = logging.getLogger(__name__)
```

Comment:
Declaring a log this high up feels premature. Why not finish the imports and then create the module log?